packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -210,43 +210,44 @@ describe('HttpMocker', () => {
       });
 
       describe('and dynamic flag is false', () => {
-        describe('and the example has been explicited', () => {
-          it('should return the selected example', () => {
+        describe('and the response has an example', () => {
+          describe('and the example has been explicited', () => {
             const response = httpMocker.mock({
               input: mockInput,
               resource: mockResource,
               config: { mock: { dynamic: false, exampleKey: 'test key' } },
             });
 
-            expect(response.body).toBeDefined();
+            it('should return the selected example', () => {
+              expect(response.body).toBeDefined();
 
-            const selectedExample = flatMap(mockResource.responses, res =>
-              flatMap(res.contents, content => content.examples || []),
-            ).find(ex => ex.key === 'test key');
+              const selectedExample = flatMap(mockResource.responses, res =>
+                flatMap(res.contents, content => content.examples || []),
+              ).find(ex => ex.key === 'test key');
 
-            expect(selectedExample).toBeDefined();
-            expect(response.body).toEqual((selectedExample as INodeExample).value);
+              expect(selectedExample).toBeDefined();
+              expect(response.body).toEqual((selectedExample as INodeExample).value);
+            });
           });
-        });
 
-        describe('and a response example is defined, but no response example is requested', () => {
-          it('returns the first example', () => {
+          describe('no response example is requested', () => {
             const response = httpMocker.mock({
               input: mockInput,
               resource: mockResource,
               config: { mock: { dynamic: false } },
             });
 
-            expect(response.body).toBeDefined();
-
-            const selectedExample = mockResource.responses[0].contents![0].examples![0];
+            it('returns the first example', () => {
+              expect(response.body).toBeDefined();
+              const selectedExample = mockResource.responses[0].contents![0].examples![0];
 
-            expect(selectedExample).toBeDefined();
-            expect(response.body).toEqual((selectedExample as INodeExample).value);
+              expect(selectedExample).toBeDefined();
+              expect(response.body).toEqual((selectedExample as INodeExample).value);
+            });
           });
         });
 
-        describe('and the resource has no response examples', () => {
+        describe('and the response has not an examples', () => {
```

Comment:
This wording was better before I think. We’re trying to clarify what type of examples it doesn’t have. response examples was meant to mean content examples or something, to avoid confusing it with property examples or something else